src/languages/groovy.js

```diff
@@ -77,25 +92,35 @@ export default function(hljs) {
                     hljs.UNDERSCORE_TITLE_MODE
                 ]
             },
-            hljs.C_NUMBER_MODE,
             {
                 className: 'meta', begin: '@[A-Za-z]+'
             },
             {
-                // highlight map keys and named parameters as strings
-                className: 'string', begin: /[^\?]{0}[A-Za-z0-9_$]+ *:/
+              // highlight map keys and named parameters as attrs
+              className: 'attr', begin: IDENT_RE + '[ ]*:'
```

Comment:
Should here be \s in place of a space?..

Reply:
See prior note, again do not want to eat new lines.

Reply:
Adding support for tabs.